data/wanderers start.txt

```diff
@@ -2435,10 +2438,13 @@ mission "Wanderers Ap'arak 3: Pug"
 	to offer
 		has "Wanderers Ap'arak 2: done"
 	npc kill
-		personality staying uninterested
+		personality staying unconstrained uninterested
 		government "Pug"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
 		ship "Pug Arfecta" "Graak Sput Kchoo"
+	on complete
+		dialog `Shortly after defeating the Pug ships, you receive a message from Sobari Tele'ek: "We are [aware?, incredulous?] of reports that you have [exterminated, eradicated] our Keepers in the Ap'arak system. This is unacceptable! From now on, you are no longer welcome in our [territory, space]."`
```

Comment:
Question-marked terms in Wanderer translations should not be followed by commas.